static/css/overrides.css

```diff
@@ -0,0 +1,75 @@
+/* Circle in navigation */
+#sidebar ul li .read-icon,
+#sidebar ul li.visited > div > .read-icon,
+#sidebar ul li.visited.active > div > .read-icon {
+  display: none;
+}
+
+/* Arrow if nav has children */
+#sidebar ul li .category-icon {
+  color: #fff;
+}
+
+/* Sidebar */
+#sidebar {
+  background-color: #333;
+  background-image: url('https://kubernetes.io/images/texture.png');
```

Comment:
My only suggestion would be to pull down the image and put it in the local `static/images` directory. That way if the other site is down or the file location changes the contributor site itself would not be impacted.

Reply:
True, I meant to do so but missed that in my final pass. I'll put it in `static/`